lib/fluent/config/dsl.rb

```diff
@@ -27,10 +28,55 @@ def self.read(path)
         end
 
         def self.parse(source, source_path="config.rb")
+          confpath = File.dirname(File.expand_path(source_path))
+          endres = parse_include(source,confpath,[])
           Proxy.new('ROOT', nil).eval(source, source_path).to_config_element
         end
+        def self.parse_include(content,confpath=nil,attrs=[])
+          attrs.push(content)
+          require 'httpclient'
```

Comment:
fluentd doesn't depend on `httpclient` gem so don't use such library in the core.